.github/workflows/api-tests.yml

```
@@ -8,41 +8,28 @@ on:
   api-tests:
-    needs: changes-check
-    if: github.event_name == 'workflow_dispatch' || github.event.action == 'opened' || (github.event.action == 'synchronize' && needs.changes-check.outputs.any_changed == 'true')
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
 
     - name: Setup Database configuration
+      shell: bash
+      env:
+        DATABASE_PASSWORD: ${{ secrets.DATABASE_PASSWORD }}
       run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)